components/director/pkg/persistence/persistence.go

```diff
@@ -45,9 +46,14 @@ type db struct {
 }
 
 func (db *db) PingContext(ctx context.Context) error {
+	db.sqlDB.Stats()
```

Comment:
What is the purpose of this line here?

Reply:
Sorry, this shouldn't be here. I will fix it in a moment.